src/styles/i-settings/_colors.scss

```diff
@@ -1,6 +1,6 @@
 /* === Neutrals === */
-$color-light: hsla(0, 0, 100, 1);
-$color-dark: hsla(0, 0, 0, 1);
+$color-light: hsla(0, 0%, 100%, 1);
```

Comment:
Sass requires you have a % sign next to saturation and lightning
https://sass-lang.com/documentation/modules/color

Reply:
According to this doc, hsla is an alias for hsl so we can remove the a for the color values https://developer.mozilla.org/en-US/docs/Web/CSS/color_value